packages/website/docs/docs/formatting-toolbar.md

```diff
@@ -18,9 +18,180 @@ The Formatting Toolbar appears whenever you highlight text in the editor, and is
 
 <img style="max-width:600px" :src="isDark ? '/img/screenshots/formatting_toolbar_dark.png' : '/img/screenshots/formatting_toolbar.png'" alt="image">
 
-## Custom Formatting Toolbar
+## Custom Formatting Toolbar Element
 
-You can create a custom Formatting Toolbar using a React component. This component should take the following props:
+You can create a custom Formatting Toolbar using a React component. The example below shows a basic custom formatting toolbar with four items. The first three are default items to toggle bold, italic, and underline, while the last one is a custom button which toggles the text and background colors to be blue.
+
+::: sandbox {template=react-ts}
+
+```typescript-vue /App.tsx
+import { BlockNoteEditor } from "@blocknote/core";
+import {
+  BlockNoteView,
+  createReactFormattingToolbarFactory,
+  ToggledStyleButton,
+  Toolbar,
+  ToolbarButton,
+  useBlockNote,
+} from "@blocknote/react";
+import "@blocknote/core/style.css";
+
+const CustomFormattingToolbar = (props: { editor: BlockNoteEditor }) => {
+  return (
+    <Toolbar>
+      {/*Default button to toggle bold.*/}
+      <ToggledStyleButton editor={props.editor} toggledStyle={"bold"} />
+      {/*Default button to toggle italic.*/}
+      <ToggledStyleButton editor={props.editor} toggledStyle={"italic"} />
+      {/*Default button to toggle underline.*/}
+      <ToggledStyleButton editor={props.editor} toggledStyle={"underline"} />
+      {/*Custom button to toggle blue text & background color.*/}
+      <ToolbarButton
+      mainTooltip={"Blue Text & Background"}
+      onClick={() => {
+        props.editor.toggleStyles({
+          textColor: "blue",
+          backgroundColor: "blue",
+        });
+      }}
+      isSelected={
+        props.editor.getActiveStyles().textColor === "blue" &&
+        props.editor.getActiveStyles().backgroundColor === "blue"
+      }>
+        Blue
+      </ToolbarButton>
+    </Toolbar>
+  );
+};
+
+export default function App() {
+  // Creates a new editor instance.
+  const editor: BlockNoteEditor = useBlockNote({
+    theme: "{{ getTheme(isDark) }}",
+  });
+
+  // Renders the editor instance.
+  return (
+    <BlockNoteView editor={editor}>
+      <FormattingToolbarPositioner
+        editor={editor}
+        formattingToolbar={CustomFormattingToolbar}
+      />
+      <HyperlinkToolbarPositioner editor={editor} />
+      <SlashMenuPositioner editor={editor} />
+      <SideMenuPositioner editor={editor} />
+    </BlockNoteView>
+  );
+}
+```
+
+```css-vue /styles.css [hidden]
+{{ getStyles(isDark) }}
+```
+
+:::
+
+Let's go over this code, starting with the `CustomFormattingToolbar` component, which is used to render the toolbar and takes the following props:
```

Comment:
I'm in doubt whether any of the docs below this are necessary. I'm starting to think that we benefit more from keeping the docs simple and focus on the core functionality, combined with a clear code base and some examples that we can point people to.